package-lock.json

```
@@ -1,6 +1,6 @@
 {
   "name": "EditorConfig",
-  "version": "0.14.2",
+  "version": "0.14.4",
```

Why are we bumping 2 patch versions here?

---

Actually, this version wasn't in sync. Should be bumped up to this:
```suggestion
  "version": "0.14.5",
```